src/Slider.jsx

```diff
@@ -8,11 +8,20 @@ import * as SliderConstants from './constants/SliderConstants';
 import linear from './algorithms/linear';
 
 function getClassName(props) {
-  const orientation = props.orientation === 'vertical'
+  const orientationClassName = props.orientation === 'vertical'
     ? 'rheostat-vertical'
     : 'rheostat-horizontal';
+  const classNames = ['rheostat', orientationClassName];
 
-  return ['rheostat', orientation].concat(props.className.split(' ')).join(' ');
+  if (props.disabled) {
+    classNames.push('rheostat-disabled');
+  }
+
+  if (props.className) {
+    classNames.push(...props.className.split(' '));
```

Comment:
I still think this should use the `cx` package instead of `split`.

Reply:
I haven't introduced manual splitting, it had been there before. Probably it's worth doing that but looks like it's out of scope of this pr.

Reply:
Fair enough.